### DISCUSSION TITLE
Add 4 unused paintings

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#1036] [__Dismissed__] [answered]
Mojang added 4 new paintings in 22w16a (1.19) earth, fire, water, and wind. These are unused and can only be summoned through commands.

It would be cool if there was a config option to add these paintings to the normal list of painting, so people could use these just like any other painting.

**Comment by encode42** [comment, marked as answer]
Easily done with a [datapack](https://www.planetminecraft.com/data-pack/all-paintings-available/).
